src/controller/MovieController.ts

```diff
@@ -103,4 +103,31 @@ export class MovieController {
       res.status(statusCode).json(result);
     }
   }
+
```

Comment:
Vamos adicionar o **Swagger-JSDoc** nesta rota também!
Vamos precisar validar uma palavra-chave enviada na requisição, se for uma rota GET ela vai vir pelo queryParam.
Temos um [middleware](https://github.com/matheus-yudi-newrizon/review-reveal-api/blob/develop/src/middleware/ValidateQuery.ts) pra verificar queryParam. Seguir este [exemplo](https://github.com/matheus-yudi-newrizon/review-reveal-api/blob/34933d2357aca6afcded5fcdc5ce651e1047f88d/src/routes.ts#L22).